cli/user/src/test/java/io/pravega/cli/user/config/InteractiveConfigCommandTest.java

```diff
@@ -54,4 +57,28 @@ public void testSetConfig() {
         interactiveConfig.set(InteractiveConfig.LOG_LEVEL, "debug");
         Assert.assertEquals(interactiveConfig.getLogLevel(), Level.DEBUG);
     }
+
+    @Test
+    public void testGetDefaultsForTlsEnv() throws NoSuchFieldException, IllegalAccessException {
+        Map<String, String> unmodifiableEnv = System.getenv();
+        Field field = unmodifiableEnv.getClass().getDeclaredField("m");
+        field.setAccessible(true);
+        Map<String, String> modifiableEnv = (Map<String, String>) field.get(unmodifiableEnv);
+        modifiableEnv.put("PRAVEGA_CONTROLLER_URI", "tls://testControllerURI");
+        InteractiveConfig interactiveConfig = InteractiveConfig.getDefault();
+        Assert.assertEquals("testControllerURI", interactiveConfig.getControllerUri());
+        Assert.assertEquals(true, interactiveConfig.isTlsEnabled());
+    }
+
+    @Test
+    public void testGetDefaultsForNonTlsEnv() throws NoSuchFieldException, IllegalAccessException {
+        Map<String, String> unmodifiableEnv = System.getenv();
```

Comment:
first 4 lines of code blocks are same in both newly added tests. Both of them can be clubbed after `Assert.assertEquals` of first method.